### PR TITLE
Update sdk 13 37

### PR DESCRIPTION
**Pull request by scealiontach**
Updates SDK to 13.37
* adds a postgres database to yamls for the index server
* reduces concurrency on daml-tp while we resolve the zmq threading issues.
* removes the tracer-ui from the daml-local.yaml since going forward this won't necessarily be sourced from this build

### CODE REVIEW

File: docker/ledger-api-testtool.docker [hunk 1]
```diff
@@ -13,8 +13,8 @@
 # limitations under the License.
 FROM ubuntu:bionic
 
-ARG DAML_SDK_VERSION=0.13.21
-ARG DAML_REPO_VERSION=100.13.21
+ARG DAML_SDK_VERSION=0.13.37
+ARG DAML_REPO_VERSION=100.13.36
```
Comment (dasormeter) [inline docker/ledger-api-testtool.docker]: @scealiontach shouldn't these both be 37?

Comment (dasormeter) [reply docker/ledger-api-testtool.docker]: path to test tool in bintray changed slightly : new path is
`https://bintray.com/api/v1/content/digitalassetsdk/DigitalAssetSDK/com/daml/ledger/testtool/ledger-api-test-tool/${sdkVersion}/ledger-api-test-tool-${sdkVersion}.jar?bt_package=sdk-components`

Comment (scealiontach) [reply docker/ledger-api-testtool.docker]: resolved

**Comment by scealiontach** [comment]
So I took the opportunity to expand the integration testing battery to include everything but the following suites:

- LotsOfPartiesIT - excluded since tests in there doesn't respect the concurrent runs, and just blast a ton of concurrent requests, which though they get processes, cannot currently be processed in time to pass.  
- PartyManagementServiceIT - Currently returning INTERNAL: None.get on some tests.  Not particularly clear what that means.
- PackageManagementService - test appears to fail because of overly ambitious concurrency goals like LotsOfPartiesIT
- TimeIT - this fails because of the "set time" test, which we don't support
- TransactionServiceIT - mostly succeeds, but certain tests report "RESOURCE_EXHAUSTED" and thats all we get so far.  the message isn't familiar to me so it will need to be investigated.

**Pull request merged**